pkg/virt-handler/cmd-client/client.go

```diff
@@ -129,24 +129,22 @@ func ListAllSockets() ([]string, error) {
 		return nil, err
 	}
 
-	if exists == false {
-		return socketFiles, nil
-	}
-
-	files, err := os.ReadDir(socketsDir)
-	if err != nil {
-		return nil, err
-	}
-	for _, file := range files {
-		if file.IsDir() || !strings.Contains(file.Name(), "_sock") {
-			continue
+	if exists {
```

Comment:
`It("Listing all sockets", func() {` would be a good place for a test

Reply:
Added unit tests for 
1) list all sockets if both legacy sockets and new sockets exist
2) list all sockets if no legacy sockets directory exists.